src/main/pages/che-7/overview/proc_installing-che-on-openshift-3-using-the-operator.adoc

```diff
@@ -15,6 +15,7 @@ This section describes how to install {prod-short} on OpenShift 3 with the CLI m
 * The `oc` OpenShift 3.11 CLI management tool is installed and configured. See link:https://docs.openshift.com/container-platform/3.11/cli_reference/get_started_cli.html#installing-the-cli[Installing the OpenShift 3.11 CLI].
 * The `{prod-cli}` CLI management tool is installed. See link:{site-baseurl}che-7/installing-the-{prod-cli}-management-tool/[Installing the `{prod-cli}` CLI management tool].
 * The URL for applications on this OpenShift instance is known as `<OPENSHIFT_APPS_URL>`. See xref:preparing-openshift-3-for-installing-che_{context}[].
+* Prepare a YAML file `configuration.yaml` containing the configuration options. See link:{site-baseurl}che-7/configuring-the-che-installation[Configuring the {prod-short} installation].
```

Comment:
I would make this step optional. Only if the user wants to use settings that cannot be set directly by the main chectl comand line parameters.

Reply:
```suggestion
*  To apply settings that can't be set directly by the main {prod-cli} command line parameters, prepare a YAML file `configuration.yaml` containing the configuration options. See link:{site-baseurl}che-7/configuring-the-che-installation[Configuring the {prod-short} installation].
```